index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desafio dos diamantes</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <select name="" id="select_diamante">
```

Comment:
```suggestion
    <select id="select_diamante">
```

Lelé, essa parte do código não precisa existir, tendo em vista que o name está vazio.

O que voce acha sobre isso?

Reply:
retirado name da linha 11